tv_apps/tv-ftu/js/tv_ftu.js

```diff
@@ -47,26 +105,36 @@ window.addEventListener('load', function() {
     },
 
     onKeyMove: function (key) {
-      switch (key) {
-        case 'up':
-          if (this._currentFocused !== this._termsSection) {
-            this.buttonNav.blur();
-            this.buttonNav.pause();
-            this._termsSection.focus();
-            this.fire('focus', this._termsSection);
-            this._arrowIcon.classList.remove('hidden');
+
+      switch (this._currentNavigationTarget) {
+        case 'unscrolled_content':
+          if (key === 'down' && this.getContentUnscrolledHeight() <= 0) {
+            this.changeNavigationTarget('buttons');
+          }
+        break;
+
+        case 'links':
+          // this.keyNav only handles
+          // the left/right key (horizontal key navigation)
+          // so we need extra handling of the up/down key here
+          if ((key === 'down' || key === 'right') &&
+            'moz-privacy-policy-link' === this._currentFocused.id) {
+            // If focus on the last link, moz-privacy-policy-link,
+            // then change navigation target to buttons
```

Comment:
After some discussion I realize this decision has been discussed with UX, then I'm ok with it.